UI/css/style.css

```diff
@@ -128,21 +132,27 @@ footer {
      padding-bottom: 5px;
      min-height: 10px;
      font-weight: bold;
-     color: black;
+     color: white;
+     background:none;
+}
+.nav-footer ul li {
+     background: none;
 }
 .nav-footer ul li a {
      list-style: none;
-     color: sienna;
+     color: rgba(231, 172, 144, 0.89);
      cursor: pointer;
+     background: none;
 }
 
 footer nav ul li a {
      text-decoration: none;
+     background: none;
```

Comment:
What are you trying to achieve here by `ul li a`? Nesting HTML element styling brings a lot of complexity. Please explain this.

Reply:
I wanted to specify the link which I have in footer